mlir/lib/Target/LLVMIR/Dialect/LLVMIR/LLVMIRToLLVMTranslation.cpp

```diff
@@ -367,6 +409,14 @@ static LogicalResult setIntelReqdSubGroupSizeAttr(Builder &builder,
   return success();
 }
 
+namespace mlir {
+// Generate llvm.call_intrinsic when no supporting dialect available.
+LogicalResult LLVMImportInterface::convertUnregisteredIntrinsic(
+    OpBuilder &builder, llvm::CallInst *inst,
+    LLVM::ModuleImport &moduleImport) const {
+  return convertUnregisteredIntrinsicImpl(builder, inst, moduleImport);
+}
+} // namespace mlir
```

Comment:
```suggestion
LogicalResult mlir::LLVMImportInterface::convertUnregisteredIntrinsic(
    OpBuilder &builder, llvm::CallInst *inst,
    LLVM::ModuleImport &moduleImport) const {
  return convertUnregisteredIntrinsicImpl(builder, inst, moduleImport);
}
```
nit: We usually prefix the function definition with the namespace in this case. The comment can be omitted since it is already in the header.